gnucash/report/reports/example/sample-report.scm

```diff
@@ -0,0 +1,485 @@
+            (gnc:html-document-add-object! document txt)))
+      
+      ;; here's a bullet list of accounts.  We can mark up the
+      ;; account name with an <a></a> anchor with a special HREF to
+      ;; open a Gnucash register when the link is clicked.  What you
+      ;; need to do is pass the HREF "gnc-register:account=My
+      ;; Account Name" to html-markup-anchor.  The account name
+      ;; passed must be the "full" account name that you get from
+      ;; gnc-account-get-full-name.  You should build this url using
+      ;; (gnc-build-url ...)
+      ;;
+      ;; html-markup-anchor takes the link to jump to as its first
+      ;; arg and then puts the remaining args in the body of the
+      ;; link).
+      ;;
+      ;; html-markup-ul makes a "<ul>" unnumbered list, and takes as
+      ;; its one argument a list of items to put in <li> blocks.
+      (if (not (null? accounts))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-ul
+             (map 
+              (lambda (acct)
+                (gnc:html-markup-anchor 
+                 (gnc-build-url URL-TYPE-REGISTER
+                   (string-append "account=" 
+                     (gnc-account-get-full-name
+                        acct))
+                     "")
+                 (xaccAccountGetName acct)))
+              accounts))))
+          (gnc:html-document-add-object!
+           document
+           (gnc:make-html-text
+            (gnc:html-markup-p (G_ "You have selected no accounts.")))))
+            
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-anchor (gnc-build-url URL-TYPE-HELP "gnucash-guide" "") (G_ "Display help"))))
+
+      (gnc:html-document-add-object! 
+       document 
+       (gnc:make-html-text 
+        (gnc:html-markup-p (G_ "Have a nice day!"))))
+      
+      document)))
+
+;; ------------------------------------------------------------------
+;; Here we define the actual report with gnc:define-report
+;; ------------------------------------------------------------------
+(gnc:define-report
+ 
+ ;; The version of this report.
+ 'version 1
+ 
+ ;; The name of this report. This will be used, among other things,
+ ;; for making its menu item in the main menu. You need to use the
+ ;; untranslated value here!
+ 'name (N_ "Options Example Report")
+
+ ;; The GUID for this report. This string should be unique, set once
+ ;; and left alone forever after that. In theory, you could use any
+ ;; unique string, even a meaningful one (!) but its probably best to
+ ;; use a true uuid. Get them from `uuidgen | sed -e s/-//g` and paste
+ ;; the results in here. You must make a new guid for each report!
+ 'report-guid "af02e925d0484745afb04f16e0524e87"
+
+ ;; The name in the menu
+ ;; (only necessary if it differs from the name)
+ 'menu-name (N_ "Options Example")

```

Comment:
Can you reply to first commit comment edit?
menu-name (N_ "Sample Options Report")

Reply:
> Can you reply to first commit comment edit?

No, because you didn't attach it to a code snippet like this one. Gitlab allows threading replies unattached comments, Github doesn't. What's more if you change the commented-on code snippet with a force-push Github will disable further comments on the thread.

"Options" has meaning in investing so it might be better to qualify it as "Report-options".  "Sample report-options report" seems a bit muddled to me, how about "Report-options Sample Report"?

Reply:
Or "…Example Report".

Reply:
Done as _Sample Report_ to fit in with _Sample Graphs_.